src/types.ts

```diff
@@ -100,6 +100,8 @@ export type DefaultQueryStringMap = {
   sortBy: 'sortBy';
   sortOrder: 'sortOrder';
   section: 'section';
+  filterName: 'filterName'; // for breadcrumb redirect
+  filterValue: 'filterValue';
```

Comment:
Let's drop these. They aren't really going to be part of the query string parameters, and we've already defined them in the `RequestConfigs` type